the .NOTPARALLEL above fails to serialise these, and they use the same
manifest.xml file so this fails in the middle without some retries
--- a/src/plugins/contrib/SmartIndent/Makefile.am
+++ b/src/plugins/contrib/SmartIndent/Makefile.am
@@ -101,32 +101,32 @@
 
 .NOTPARALLEL: $(pkgdata_DATA)
 
-SmartIndentHDL.zip: manifest_hdl.xml
+SmartIndentHDL.zip: manifest_hdl.xml SmartIndentPascal.zip
 	PWD=`pwd` cd $(srcdir) && \
 	cp manifest_hdl.xml ./manifest.xml && \
 	zip -R $(PWD)/SmartIndentHDL.zip manifest.xml
 
-SmartIndentPascal.zip: manifest_pascal.xml
+SmartIndentPascal.zip: manifest_pascal.xml SmartIndentCpp.zip
 	PWD=`pwd` cd $(srcdir) && \
 	cp manifest_pascal.xml ./manifest.xml && \
 	zip -R $(PWD)/SmartIndentPascal.zip manifest.xml
 
-SmartIndentCpp.zip: manifest_cpp.xml
+SmartIndentCpp.zip: manifest_cpp.xml SmartIndentPython.zip
 	PWD=`pwd` cd $(srcdir) && \
 	cp manifest_cpp.xml ./manifest.xml && \
 	zip -R $(PWD)/SmartIndentCpp.zip manifest.xml
 
-SmartIndentPython.zip: manifest_python.xml
+SmartIndentPython.zip: manifest_python.xml SmartIndentLua.zip
 	PWD=`pwd` cd $(srcdir) && \
 	cp manifest_python.xml ./manifest.xml && \
 	zip -R $(PWD)/SmartIndentPython.zip manifest.xml
 
-SmartIndentLua.zip: manifest_lua.xml
+SmartIndentLua.zip: manifest_lua.xml SmartIndentFortran.zip
 	PWD=`pwd` cd $(srcdir) && \
 	cp manifest_lua.xml ./manifest.xml && \
 	zip -R $(PWD)/SmartIndentLua.zip manifest.xml
 
-SmartIndentFortran.zip: manifest_fortran.xml
+SmartIndentFortran.zip: manifest_fortran.xml SmartIndentXML.zip
 	PWD=`pwd` cd $(srcdir) && \
 	cp manifest_fortran.xml ./manifest.xml && \
 	zip -R $(PWD)/SmartIndentFortran.zip manifest.xml
